modules/extensions-xml-support/pom.xml

```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
this should be added as a dependency in the all-modules pom. I keep getting the feeling you didn't try to build a local distro